cmd/controller/controller-runtime-main.go

```diff
@@ -60,19 +82,31 @@ func controllerRuntimeStart() error {
 		feedsv1alpha1.AddToScheme,
 		flowsv1alpha1.AddToScheme,
 		istiov1alpha3.AddToScheme,
-		subscriptionsv1alpha1.AddToScheme,
+		eventingv1alpha1.AddToScheme,
 	}
 	for _, schemeFunc := range schemeFuncs {
 		schemeFunc(mrg.GetScheme())
 	}
 
 	// Add each controller's ProvideController func to this list to have the
-	// manager run it.
-	providers := []ProvideFunc{
-		eventtype.ProvideController,
-		feed.ProvideController,
-		flow.ProvideController,
-		subscription.ProvideController,
+	// manager run it by config.
+	providers := make([]ProvideFunc, 0, len(KnownProvideControllers))
```

Comment:
If you're not concerned about performance, you can write
```golang
providers := []ProvideFunc{}
```

Reply:
I know, and I am :D